docs/modules/ROOT/pages/sota-variables.adoc

```diff
@@ -0,0 +1,50 @@
+= Build Configuration Options
+:page-partial:
+// MC: Included in aktualizr/docs/ota-client-guide/modules/ROOT/pages/build-configuration.adoc
+
+.OTA-related options for building disk images
+[cols="1,2a",options="header"]
+|====================
+| Option | Description
+| `OSTREE_BRANCHNAME`|OSTree branch name. Defaults to `${SOTA_HARDWARE_ID}`. Particularly useful for grouping similar images.
+| `OSTREE_REPO`|Path to your OSTree repository. Defaults to `$\{DEPLOY_DIR_IMAGE}/ostree_repo`
+| `OSTREE_OSNAME`|OS deployment name on your target device. For more information about deployments and osnames see the https://ostree.readthedocs.io/en/latest/manual/deployment/[OSTree documentation]. Defaults to "poky".
+| `OSTREE_COMMIT_BODY`|Message attached to OSTree commit. Empty by default.
+| `OSTREE_COMMIT_SUBJECT`|Commit subject used by OSTree. Defaults to `Commit-id: ${IMAGE_NAME}`
+| `OSTREE_UPDATE_SUMMARY`|Set this to '1' to update summary of OSTree repository on each commit. '0' by default.
+| `OSTREE_DEPLOY_DEVICETREE`|Set this to '1' to include devicetree(s) to boot
+| `GARAGE_SIGN_AUTOVERSION`|Set this to '1' to automatically fetch the last version of the garage tools installed by the aktualizr-native. Otherwise use the fixed version specified in the recipe.
+| `GARAGE_TARGET_URL` | Sets the `--url` parameter of `garage-sign targets add`, which sets a custom URL for the Image repository targets.
+ifndef::env-github[]
+[NOTE]
+====
+Currently, this only works when using the master branch of meta-updater.
+====
+endif::[]
```

Comment:
Is this note for `GARAGE_TARGET_URL`? That exists in all supported branches.

Reply:
oops - No, it was intended for the expiry parameters - will fix

Reply:
fixed